oaknational/Oak-Web-Application
# fix: Downloads text selectable

## Description

- removes default of box border $zIndex (setting it neutral still gave it precedence over not set)
- about/board downloads text now selectable
- sets zindex on the lines themselves rather than the container, so that it doesn't block touch events on box contents even when zindex applied
- 


## How to test

1. Go to {cloud link} about/board
2. Try select downloads text
3. You should b able to select it 
4. 
## Checklist
